PROJECT_LIFECYCLE.md

```diff
@@ -38,6 +38,9 @@ their ability.
 - names of initial maintainers
 - infrastructure needs such as Git repositories
 
+Regarding project naming convenctions, the eiffel community enforces no name convention but meaningful descriptive 
```

Comment:
```suggestion
Regarding project naming conventions, the Eiffel Community enforces no name convention but meaningful descriptive 
```

typo + capitals

Reply:
as mentioned on Slack, I do not have a strong opinion about this so I vote 0/abstain from voting. ;)